### ISSUE TITLE
axios as devDependency

### ISSUE THREAD

**Issue by oprogramador** [js-data/js-data-http#65]
I noticed axios belongs here to devDependencies but IMO it's a prod dependency.

**Comment by Hagith** [comment]
Currently `axios` is bundled inside `dist/*.js`. Probably that's why listed only as a dev dependency.
IMHO `axios` should be `peerDependency` and not bundled into `dist`.

**Comment by oprogramador** [comment]
I consider `axios` as an implementation detail. It could be replaced by superagent or another library and behave in the same way.

So IMO it should be a prod dependency.

**Comment by jmdobry** [comment]
If you use `js-data-http`, then `axios` is bundled into `dist/js-data-http.js`. If you use `js-data-fetch`, `axios` is _not_ bundled, and you need to provide an agent, such as `superagent`. If you use `js-data-http-node`, then `axios` is a prod dependency and will be installed and loaded by your app. I will add `axios` as a prod dependency of `js-data-http`.

**Comment by chlab** [comment]
I agree with @Hagith and think axios should be a peerDependency. If you're using js-data-http but need axios in your app for other requests, it would end up being bundled twice.
